### DISCUSSION TITLE
Configure Color of Lint Errors?

### DISCUSSION THREAD

**Discussion by treatmesubj** [python-lsp/python-lsp-server#485] [Q&A] [answered]
Hi, I haven't been able to figure out how to change the text color of lint errors; is this possible?
For me, they appear as yellow text in neovim, which blends in with strings quite a bit.
I'd like to change the text to a different color.

**Comment by krassowski** [comment, marked as answer]
The LSP servers do not control the colour or other presentation details of the diagnostics. This is something you need to check with the editor you use.
